docs: Updated doc to include OVMS info (#126)
intel-retail/automated-self-checkout: docs_src/OVMS/pipelinesetup.md

# Set up Pipeline

### Step 1: Clone the repository

```
git clone  https://github.com/intel-retail/automated-self-checkout.git && cd ./automated-self-checkout
```

> ### Step 2: Build the benchmark images (coming up)

Reviewer: I prefer to call out the Docker images as there is also picture `image` we are downloading as part of steps.

```suggestion
### Step 2: Build the benchmark Docker images (coming up)
```

> ### Step 2: Build the benchmark containers (coming up)

Reviewer: ```suggestion
### Step 2: Build the benchmark images (coming up)
```

> For use cases that support benchmarking, build the benchmark images:

Reviewer: ```suggestion
For use cases that support benchmarking, build the benchmark Docker images:
```

> For use cases that support benchmarking, build the benchmark containers:

Reviewer: ```suggestion
For use cases that support benchmarking, build the benchmark Docker images:
```

```bash
cd benchmark-scripts
make build-all
```

### Step 3: Download the models manually (Optional)

When `docker-run.sh` the Model downloader script is automatically called. the model downloader script is automatically called as part of docker-run-opencv-ovms.sh (part of docker-run.sh). You can also download the models manually using the model downloader script:

```bash
sh ./download_models/getModels.sh --workload opencv-ovms
```

**_Note:_**  Depending on your internet connection, this might take less than a minute.

### Step 4: Download image file Manually (Optional)

The sample image downloader script is automatically called as part of docker-run-opencv-ovms.sh. You can also download the sample image manually using script below:

```bash
sh ./configs/opencv-ovms/scripts/image_download.sh 
```

> **_Note:_** Depending on your internet connection, this might take less than a minute.

Reviewer 2: ```suggestion
**_Note:_** Depending on your internet connection, this might take longer than a minute.
```

| ----------------------------------| -----------------------------------|------------------------------------------------------------------------|
| OVMS Server                       | <pre>make build-ovms-server</pre>  | docker images command to show <b>openvino/model_server-gpu:latest</b>  |
|                                   |                                    | docker images command to show <b>openvino/model_server:latest-gpu</b>  |
|                                   |                                    | docker images command to show <b>openvino/model_server:latest</b>      |
|                                   |                                    | docker images command to show <b>openvino/model_server-pkg:latest</b>  |
|                                   |                                    | docker images command to show <b>openvino/model_server-build:latest</b>|
| OVMS Client                       | <pre>make build-ovms-client</pre>  | docker images command to show <b>ovms-client:latest</b>                |

**_Note:_** Build command may take a while, depending on your internet connection and machine specifications.

> **_Note:_** If the build command succeeds, you will see all the built Docker images files as indicated in the **Check Success** column. If the build fails, check the console output for errors. The dependencies might have been unable to resolve. Address the issue and repeat [from step 3](/pipelinesetup.md#step-3).

Reviewer: i don't understand this note... why docker image build failure has anything to do with model downloading ?

Author: build can fail if the model downloading failed, so this is asking user to check from step 3 (download models)

Reviewer: could you show me?
